packages/compass-e2e-tests/tests/read-only.test.ts

```diff
@@ -0,0 +1,421 @@
+import { beforeTests, afterTests, afterTest } from '../helpers/compass';
+import { promises as fs } from 'fs';
+import path from 'path';
+import os from 'os';
+import { expect } from 'chai';
+import * as Selectors from '../helpers/selectors';
+import { createNumbersCollection } from '../helpers/insert-data';
+
+describe('readOnly: true / Read-Only Edition', function () {
+  let tmpdir: string;
+  let i = 0;
+
+  beforeEach(async function () {
+    tmpdir = path.join(
+      os.tmpdir(),
+      `compass-global-preferences-${Date.now().toString(32)}-${++i}`
```

Comment:
```suggestion
      `compass-read-only-${Date.now().toString(32)}-${++i}`
```

Reply:
But this dir is for global preferences, right? Maybe it is ok to call it the same? I have seen it also named like this here: https://github.com/mongodb-js/compass/blob/COMPASS-6064-implement-readonly-option/packages/compass-e2e-tests/tests/no-network-traffic.test.ts#L20

So I assumed we wanted to call it preferences :)

Reply:
> I have seen it also named like this here: https://github.com/mongodb-js/compass/blob/COMPASS-6064-implement-readonly-option/packages/compass-e2e-tests/tests/no-network-traffic.test.ts#L20

Yeah that just means I was too lazy and copied it without fixing it :see_no_evil: 

The name is `compass-global-preferences-...` because it is called that way in the `global-preferences.test.ts` file. It’s used to avoid conflicts between tests (since the `i` variable is local to each test), in the rare case in which `Date.now()` might not be enough to disambiguate directories (very unlikely for e2e tests, more likely for other tests).

So, no, this is not because it’s about storing global preferences, it’s about the test name.

Reply:
Got it! :) will update two places then!